packages/wallets/src/components/JurisdictionModal/jurisdiction-contents/props.types.ts

```diff
@@ -0,0 +1,78 @@
+export type TClickableDescription = {
+    onClick?: React.MouseEventHandler<HTMLSpanElement>;
+    text: string;
+    type: 'link' | 'text';
+};
+export type TJurisdictionCardSectionTitleIndicators = {
+    displayText?: string;
+    displayTextSkinColor?: string;
+    type: 'displayIcons' | 'displayText';
+};
+export type TJurisdictionCardSection = {
+    clickableDescription?: TClickableDescription[];
+    description?: string;
+    key: string;
+    title: string;
+    titleIndicators?: TJurisdictionCardSectionTitleIndicators;
+};
+export type TJurisdictionCardVerificationStatus = 'Default' | 'Failed' | 'Pending' | 'Verified';
+export type TJurisdictionCardItemVerificationItem =
+    | 'documentNumber'
+    | 'identityDocument'
+    | 'nameAndAddress'
+    | 'notApplicable'
+    | 'selfie';
+export type TJurisdictionCardItemVerification = TJurisdictionCardItemVerificationItem[];
+export type TJurisdictionCardItems = {
+    contents: {
+        financial: TJurisdictionCardSection[];
+        all?: TJurisdictionCardSection[];
+        synthetic: TJurisdictionCardSection[];
+    };
+    header: string;
+    isOverHeaderAvailable: boolean;
+    overHeader?: string;
+    verificationDocs?: {
+        financial?: TJurisdictionCardItemVerification;
+        synthetic?: TJurisdictionCardItemVerification;
+    };
+};
+export type TJurisdictionCardParams = {
+    toggleDynamicLeverage: React.MouseEventHandler<HTMLSpanElement>;
+};
+export type TJurisdictionVerificationSection = {
+    icon: string;
+    text: string;
+};
+export type TJurisdictionVerificationItems = {
+    documentNumber?: TJurisdictionVerificationSection;
+    identityNumber?: TJurisdictionVerificationSection;
+    nameAndAddress?: TJurisdictionVerificationSection;
+    notApplicable?: TJurisdictionVerificationSection;
+    selfie?: TJurisdictionVerificationSection;
+};
+type TJurisdictionVerificationColors = 'green' | 'red' | 'yellow';
+export type TJurisdictionVerificationStatus = {
+    color: TJurisdictionVerificationColors;
+    icon: string;
+    text: string;
+};
+export type TInstrumentsIcon = {
+    className?: string;
+    highlighted: boolean;
+    icon:
+        | 'Baskets'
+        | 'Commodities'
+        | 'Cryptocurrencies'
+        | 'DerivedFX'
+        | 'ETF'
+        | 'Forex'
+        | 'StockIndices'
+        | 'Stocks'
+        | 'Synthetics';
+    isAsterisk?: boolean;
+    text: string;
+};
+export type TJurisdictionData = {
+    jurisdiction?: 'bvi' | 'labuan' | 'malta' | 'maltainvest' | 'svg' | 'vanuatu';
```

Comment:
Lets type this from BE response. We should avoid putting these type of landing_company in FE whenever possible. 

```suggestion
    jurisdiction?:  NonNullable<ReturnType<typeof useMT5AccountsList>['data']>[0]['landing_company_short'];
```
Already have whatever you need 
<img width="979" alt="Screenshot 2023-10-12 at 9 23 26 AM" src="https://github.com/binary-com/deriv-app/assets/104053934/7ad9954f-169d-443d-b8d8-6801c3c3124c">

Reply:
But the `sychelles` is no longer used so i think it will give type error in places where we have different types. So use a unified type everywhere.

Reply:
We are not using seychelles at all in wallets package. Or even in the data we are hardcoding for now. Shouldnt be a problem. 😸